src/routes/help/components.svelte

```diff
@@ -0,0 +1,238 @@
+<script>
+	import components from '../components/components.json';
+
+	const allTags = Array.from(new Set(components.map((item) => item.tags).flat()));
+	const tagItems = allTags.map((tag) => ({ label: tag, value: tag })).sort(ascending);
+	const allCategories = Array.from(new Set(components.map((item) => item.category).flat()));
+	const categoryItems = allCategories
+		.filter((cat) => cat !== '')
+		.map((cat) => ({ label: cat, value: cat }))
+		.sort(ascending);
+
+	let clipboardCopy = false;
+	const copyToClipboard = (text) => {
+		navigator.clipboard
+			.writeText(text)
+			.then(() => {
+				clipboardCopy = true;
+				setTimeout(() => (clipboardCopy = false), 1000);
+			})
+			.catch(() => alert('Clipboard copy Permission denied'));
+	};
+
+	let title = 'svelte-calender';
+	let url = 'https://github.com/6eDesign/svelte-calendar';
+	let description = 'A lightweight date picker with neat animations and a unique UX';
+	let npm = 'svelte-calender';
+	let image = '';
+	let category = 'Forms & User Input';
+	let tags = ['components and libraries', 'time and date'];
+	let addedOn = '';
+
+	$: jsonSnippet = {
+		title,
+		url,
+		description,
+		npm,
+		tags,
+		image,
+		addedOn,
+		category
+	};
+
+	function ascending(a, b) {
+		return a.label > b.label ? 1 : -1;
+	}
+</script>
+
+<h1>How to submit a new component?</h1>
```

Comment:
Some headings are questions and some are not. Let's make none of them questions for consistency
```suggestion
<h1>Submitting a new component</h1>
```